### PR TITLE
Created tasks' templates for feature request and bug report

### PR DESCRIPTION
**Pull request by ErnestaP**
ref : https://github.com/orgs/cern-sis/projects/1/views/7

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            <td>Proposed solution(s): describe technical solutions</td>
+        </tr>
+        <tr>
+            <td>Expected behaviour</td>
+        </tr>
+         <tr>
+            <td>Origin: where does the request come from? (Product owner, user feedback, etc)</td>
+        </tr>
+        <tr>
+            <td>Tests required? Remember to test new code to ensure the quality.</td>
```
Comment (ParthS007) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
            <td>Tests required? Remember to test new code to ensure the quality.</td>
            <td>Mention any edge cases that will be useful for testing.</td>
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <tbody>
+        <tr>
+            <td rowspan=6>A high-level summary of the request</td>
+            <td>Requirement: why is this feature needed? </td>
+            <td rowspan=6>Add labels to indicate what the task is about. A label can also be used to flag tasks for refinement. </td>
+            <td rowspan=6>Low, medium, high, blocker</td>
+            <td rowspan=6>Determine assignees before the upcoming sprint</td>
+        </tr>
+        <tr>
+            <td>Proposed solution(s): describe technical solutions</td>
```
Comment (karolina-siemieniuk-morawska) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I have one question/suggestion/doubt - feature requests can be made not only (in our case, and generally even mainly) by non-technical people, so suggesting technical solutions might be an issue. Also I think that it should be on dev's side to propose a technical solution for the requested feature. What does the team think?

Comment (ErnestaP) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Agree. Also, it can be rephrased to "Proposed solution(s)"  and that's it. Because maybe people who are questioning the feature know what has to be done, but not in tech. way. And we, as developers, during refinement can add more details

**Comment by drjova** [comment]
@ErnestaP thank you! There is also this https://github.com/cern-sis/workflows/issues/templates/edit that help you build the template.
